src/components/Login.js

```diff
@@ -1,17 +1,18 @@
 import React, { useState, useEffect } from 'react';
 import { useNavigate} from 'react-router-dom';
 import axios from 'axios';
-import "./LoginRegister.css" 
+import "./LoginRegister.css"
 
 const Login = ({currentUser, setCurrentUser, setIsLoggedIn, token}) => {
-    console.log('In login user!');
     const [username, setUsername] = useState("");
     const [password, setPassword] = useState("");
+    const [showCredentialsError, setShowCredentialsError] = useState(false);
+    const [loginError, setLoginError] = useState("");
     const navigate = useNavigate();
 
     const loginUser = () => {
 
-        if (!username && !password) {
+        if (!username || !password) {
```

Comment:
looks good! This could even be on one line: `if (!username || !password) return;`

Reply:
updated and resolved.